2-Browsers/Week1/homework/ex2-aboutMe/index.js

```diff
@@ -8,4 +8,17 @@
    `list-item`.
 ------------------------------------------------------------------------------*/
 
-// TODO add your JavaScript code here.
+
+const body = document.querySelector("body");
+const nickName = document.getElementById("nickname");
+const favFood = document.getElementById("fav-food");
+const homeTown = document.getElementById("hometown");
+const listItem = document.getElementsByTagName("li");
+body.style.fontFamily = "Arial, sans-serif";
+nickName.innerText = "Ran";
+favFood.innerText = "lasagna";
+homeTown.innerText = "Damascus";
+
+listItem.forEach(item => item.className = "list-item");
```

Comment:
What would happen if there already exists a class on the `listItem` ?

Reply:
It will replace the class name with the new one, so if there is already a class name then we can add another class name using element.classList.add( ).